README.md

```diff
@@ -24,10 +24,31 @@ using Statistics
 lazfn = joinpath(dirname(pathof(LazIO)), "..", "test/libLAS_1.2.laz")
 pointcloud = LazIO.open(lazfn)
 
+#LAS file support is provided through LazIO.open() as well
+pointcloud = LazIO.open(somefile.las)
```

Comment:
It would be nice if all commands just work if users copy paste them. This line won't work, it is not a string and that file doesn't exist. Maybe the comment can just go above the other `LazIO.open`?

Reply:
OK, I agree. I think the comment captures what I was trying to explain well enough.